### DISCUSSION TITLE
Provide Me the Sample of Letter to GitHub for GitHub Education.

### DISCUSSION THREAD

**Discussion by santoshvandari** [community/community#69780] [GitHub Education] [answered]
### Hi everyone,

Hello Everyone

I am currently studying in Bachelor's Level. I tried to verify my GitHub education using an ID card, But It doesn't. So I go for the second one. 
![image](https://github.com/community/community/assets/63558580/adae2495-28d6-49fb-957c-b99f4081134d)

" If you don't have a student ID, or it doesn't include a date, you can upload a letter on school letterhead or any documentation with a date that demonstrates your current enrollment."
I asked the college to provide me with the letter in their official letter pad for GitHub Education. Then, ask me for a Sample of the letter. Can anyone provide me with a sample of that letter? So that I can provide that sample to the college, and then they will provide me the letter by taking the reference from the letter. 
I will be very thankful. 

### A suggestion
I will be thankful if you provide me the photo of the sample of letter.

**Comment by andrej-kocijan** [comment]
You should ask your university to provide you with a "proof of enrollment" or even a transcript of your grades / passed exams, worked for me :)

**Reply by santoshvandari** [reply]
College Is asking me for the sample of Letter. Can You Provide me?

**Comment by VedasreeM** [comment]
> ### Hi everyone,
> Hello Everyone
> 
> I am currently studying in Bachelor's Level. I tried to verify my GitHub education using an ID card, But It doesn't. So I go for the second one. ![image](https://user-images.githubusercontent.com/63558580/274238876-adae2495-28d6-49fb-957c-b99f4081134d.png)
> 
> " If you don't have a student ID, or it doesn't include a date, you can upload a letter on school letterhead or any documentation with a date that demonstrates your current enrollment." I asked the college to provide me with the letter in their official letter pad for GitHub Education. Then, ask me for a Sample of the letter. Can anyone provide me with a sample of that letter? So that I can provide that sample to the college, and then they will provide me the letter by taking the reference from the letter. I will be very thankful.
> 
> ### A suggestion...(optional)
> _No response_

[Your College's Name]
[College Address]
[City, State, ZIP Code]
[Date]

Dear Sir/Madam,

I am writing to request a letter from [Your College's Name] confirming my current enrollment as a student. I am a student in the [Degree Program] and am currently in [Year/Semester]. My expected date of graduation is [Expected Graduation Date].

I need this letter for the purpose of GitHub Education verification.

Kindly consider my request.

Sincerely,

[Your Full Name]
[Your Student ID]
[Your Contact Information (optional)]

**Reply by santoshvandari** [reply]
I apologize that you haven't understood my request well. I am asking letter sample for the college not for me to request the college for the letter. 
The college is asking me to provide the format and they will write the letter according to that format.  I am asking for the letter sample for the college. If you have some then provide me.

**Comment by VedasreeM** [comment, marked as answer]
I apologize for misunderstanding your request earlier.
Here's the format:
[Your College's Name]
[College Address]
[City, State, ZIP Code]
[Date]

To Whom It May Concern,

This is to certify that [Student's Full Name] is currently enrolled as a student in the [Degree Program] at [Your College's Name]. [He/She] is in [Year/Semester] and is expected to graduate on [Expected Graduation Date].

We are providing this letter in response to [Student's Name]'s request for confirmation of enrollment for GitHub Education verification.

If you have any further questions or require additional information, please do not hesitate to contact us.

Sincerely,

[Authorized Signatory]
[Position]
[Contact Information]
